server/src/test/java/org/elasticsearch/common/util/concurrent/EsExecutorsTests.java

```diff
@@ -120,7 +121,7 @@ public void testFixedRejected() throws Exception {
             1,
             EsExecutors.daemonThreadFactory("test"),
             threadContext,
-            randomBoolean()
+            randomFrom(TaskTrackingConfig.DEFAULT, TaskTrackingConfig.DO_NOT_TRACK)
```

Comment:
nit Seems like you can refactor this in a function. Or make the DEFAULT/DO_NOT_TRACK enum values and select a random enum value.